build/deploy/mkimage.sh

```diff
@@ -12,26 +12,35 @@
 
 set -euo pipefail
 
-cd -P "$(dirname $0)"
-DIR=$(pwd -P)
+# This is mildly trick: This script runs itself recursively. The first
+# time it is run it does not take the if-branch below and executes on
+# the host computer. It uses the builder.sh script to run itself
+# inside of docker passing "docker" as the argument causing the
+# commands in the if-branch to be executed within the docker
+# container.
+if [ "${1:-}" = "docker" ]; then
+    time make testbuild
```

Comment:
I don't think so; even without the build time there's a non-trivial amount of complexity devoted to getting these tests built and running in a strange artificial environment.